roles/edpm_nova/defaults/main.yml

```diff
@@ -28,3 +28,9 @@ edpm_nova_image_download_retries: 5
 edpm_nova_config_src: /var/lib/openstack/configs
 edpm_nova_config_dest: /var/lib/openstack/config/nova
 edpm_nova_compute_image: "quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified"
+
+# certs
+edpm_nova_certs_src: /var/lib/openstack/certs
+edpm_nova_certs_dest: /var/lib/openstack/certs/nova
+edpm_nova_cacerts_src: /var/lib/openstack/cacerts
+edpm_nova_cacerts_dest: /var/lib/openstack/cacerts/nova
```

Comment:
jsut some notes:
both src dirs are paths in the AEE pod
both dest dirs are paths on the EDPM host.

we use a single AEE pod for all hosts in the OpenStack dataplane nodeset

we do not want the copy the certs for all hosts in the nodeset to each host 
so we are going to need to make sure we only copy the relevant certs for this host when we use these later

Reply:
you shoudl add

edpm_nova_tls_certs_enabled:"{{ tls_certs_enabled | default(False) }}"

Reply:
done